### ISSUE TITLE
 [question] What to do after training and testing aae only

### ISSUE THREAD

**Issue by bigbigdinosaur** [DLR-RM/AugmentedAutoencoder#22]
[question]
hello!
your work is quite wonderful!
I would like to use a camera to take a picture of an object(just one kind of object),then get its 6d information through your nice work. Following the readme, i have managed _**testing Augmented Autoencoder only**_, code is:**_python aae_image.py_**. input a picture,output a SO3 rotation matrix, work properly. After that i feel confused about what to do next to get the total 6d information, should i choose one of these two methods?
 **_Option 1_**: Train a RetinaNet Model from 
**_Option 2_**: Using the Google Detection API with Fixes. 
I am new to this area and have few skill, could you please tell me which one is easy to realize? in my case only one kind object need to be detected so i think the detect network is no necessary to be complicated. 
Another question is during the training of detect network, can i use **_generate_sixd_train.py_** this code to generate training data instead of marking labels by hand?
In short, 
**_1:which option is easy 
2:whether use generate_sixd_train.py_**
As you see, my English is quite poor, but I sincerely appreciate your help and kindness!

**Comment by MartinSmeyer** [comment]
Thank you!
I would go with Option 1 and train a RetinaNet Model, even though it is a bit of an overkill for a single object. But the interface is cleaner and easier to use and so
The generate_sixd_train.py is meant to generate training data only for the SIXD datasets (like T-LESS etc.). If you have a good 3d model, you can use generate_syn_det_train.py to generate training data for the detector. Otherwise labeling 100-200 images by hand and fine-tuning RetinaNet should work as well.
